src/core_modules/capture-core/components/ListView/ListWrapper/EventsListWrapper.component.js

```diff
@@ -2,19 +2,19 @@
 import * as React from 'react';
 import { withStyles } from '@material-ui/core/styles';
 
-import elementTypes from '../../../../../metaData/DataElement/elementTypes';
+import elementTypes from '../../../metaData/DataElement/elementTypes';
```

Comment:
Maybe a NIT. I have mentioned it above. It looks to me that this can be imported from the `metadata` folder as `dataElementTypes`.

Reply:
This file is removed later.